### PR TITLE
Add information about trait implementors

### PR DESCRIPTION
**Pull request by hug-dev**
Adds a page per trait and describe the implementors that we have and
that we use.

Signed-off-by: Hugues de Valon <hugues.devalon@arm.com>

### CODE REVIEW

File: src/parsec_service/key_id_managers.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Key ID Managers
+
+# On-Disk Key ID Manager
+
+The on-disk key ID manager stores the mapping between key triple and key ID directly on disk, in a
+folder which path is configurable.
```
Comment (ionut-arm) [inline src/parsec_service/key_id_managers.md]: > folder with a configurable path.

or
> folder whose path is configurable.

File: src/parsec_service/listeners.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Listeners
+
+## Domain Socket Listener
+
+This listener communicates with its client using a [Unix domain
+socket](https://en.wikipedia.org/wiki/Unix_domain_socket).
+
+When launching directly from terminal, the socket path is fixed to `/tmp/security-daemon-socket`.
```
Comment (ionut-arm) [inline src/parsec_service/listeners.md]: The systemd daemon is also launched from the terminal. Maybe it's more clear to say something like 
> When running the service executable directly, ...

File: src/parsec_service/key_id_managers.md (hunk 1 above)
Comment (hug-dev) [reply src/parsec_service/key_id_managers.md]: Will change 😃 
https://youtu.be/01Dn53H2ZLw?t=70

File: src/parsec_service/listeners.md (hunk 2 above)
Comment (hug-dev) [reply src/parsec_service/listeners.md]: Yes, makes more sense!

File: src/parsec_service/key_id_managers.md (hunk 1 above)
Comment (ionut-arm) [reply src/parsec_service/key_id_managers.md]: don't hate the player, hate the game.

**Pull request merged**